modelconverter/packages/rvc4/exporter.py

```diff
@@ -57,6 +59,19 @@ def __init__(self, config: SingleStageConfig, output_dir: Path):
                 self._attach_suffix(self.input_model, "modified.onnx"),
                 self.inputs,
             )
+
+            onnx_modifier = ONNXModifier(
```

Comment:
These modifications were found to be beneficial for the RVC4 case after some tests and benchmarking @aljazkonec1 did. I think its unknown if these will improve (or worsen) the performance on the rest of the devices. Maybe @aljazkonec1 has a better view on this.

Reply:
I did compare output values between original onnx and the optimized onnx. In my limited testing, there were no differences between original onnx and the optimized onnx outputs. From the graph it was mostly operations like split, concat, reshape that were simplified and fused into other layers like conv

Reply:
Yes, the optimizations were done on the basis of what was slowing inference speeds on RVC4 devices. My understanding is that converting to RVC2 is pretty optimized by default from Intel itself and we do not have much insights into per layer analysis of models. In any case, this PR only affects add, mul, div, sub layers and fusing them into already existing Conv layers. I do not see a reason that different weights in Conv layers would affect inference speed on RVC2

Reply:
Actually RVC2/Intelalso has a [benchmarking tool](https://docs.openvino.ai/2024/learn-openvino/openvino-samples/benchmark-tool.html) which we could use to check how/if these optimizations affect the RVC2 exported model. But IMO we can leave this for a separate PR because we want to test it out first to verify.
+1 on optionally disabling modifications through config (by default I would have it on)